patterns/gitleaks/7.6.1/98-general.toml

```diff
@@ -119,18 +119,20 @@
   # Example match: "quay.io": {"auth": "token-token-token-token"
   # It also tries to ignore space around the json
   # (Note: this only works when it's all on a single line)
-  regex = '''(\\)?\"([^\"]+)?(docker|quay|openshift|openshiftapps|redhat)\.(io|com|org)([^\"]+)?(\\)?"(\s+)?:(\s+)?\{(\s+)?(\\)?\"auth(\\)?\"(\s+)?:(\s+)?(\\)?\"([^\"]{32,})(\\)?\"'''
+  regex = '''\\?\"[^\"]*?(?:docker|quay|openshift|openshiftapps|redhat)\.(?:io|com|org)[^\"]*?\\?"\s*:\s*\{[\s\S]*?\\?\"auth\\?\"\s*:\s*\\?\"([^\"]{32,})\\?\"'''
```

Comment:
I typically do `\\*` because of nested JSON. It's rare, but happens!

Reply:
Dat's big brain thinkin' right there. I'll make that tweak.